rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -564,6 +564,12 @@ def count(self, model_class, filters=None):
         count = query.order_by(None).count()  # Fastest way to count
         return count
 
+    def admin_list(self, model_class, filters=None):
```

Comment:
this does very little compared to the regular `list` method, but i suppose we'll add features (eg. joins, sort) as they're needed?

Reply:
Maybe it is better to add a parameter like "skip_permissions" to the regular `list` that says don't consider tenant and user?